src/mappers/dissolution/dissolutionRequest.mapper.ts

```diff
@@ -0,0 +1,20 @@
+import 'reflect-metadata'
+
+import { provide } from 'inversify-binding-decorators'
+
+import { DirectorRequest, DissolutionCreateRequest } from 'app/models/dto/dissolutionCreateRequest'
+import DirectorToSign from 'app/models/session/directorToSign.model'
+
+
+@provide(DissolutionRequestMapper)
+export default class DissolutionRequestMapper {
+
+  public mapToDissolutionRequest(directors: DirectorToSign[]): DissolutionCreateRequest {
```

Comment:
mapToDissolutionCreateRequest. I also think this mapper should accept the full DissolutionSession object as in future the request will contain additional fields beyond just directors. Future proofing

Reply:
^